pubtools/pulplib/_impl/client/client.py

```diff
@@ -239,7 +286,7 @@ def _unpack_response(cls, pulp_response):
             parsed = pulp_response.json()
         except Exception:
             # Couldn't parse as JSON?
-            # If the response was unsuccessful, raise that.
+            # In other cases, if the response was unsuccessful, raise that.
```

Comment:
This change doesn't make sense now, should put it back.
```suggestion
            # If the response was unsuccessful, raise that.
```